wordpress-mobile/WordPress-Android
# Issue/update themes list

### Fix
Update list view in ***Themes*** screen.  The placeholder and preview images were made full-width.  The ellipsis icons were made to match the theme name text color.  See the screenshots below for illustration.

![themes](https://user-images.githubusercontent.com/3827611/52684915-124eb200-2f05-11e9-968d-80692b121528.png)

### Test
0. Use device with slow network connection.
1. Go to ***Sites*** tab.
2. Tap ***Themes*** under ***Look and Feel*** section.
3. Notice placeholder image is full-width in list view.
4. Wait for preview image to load.
5. Notice preview image is full-width in list view.
6. Wait for active theme to be selected.
7. Notice ellipsis icon is white for active theme.
8. Notice ellipsis icon is black for inactive themes.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
